frontend/server/src/Controllers/Run.php

```diff
@@ -1008,6 +1028,18 @@ private static function getOptionalRunDetails(
         ) {
             $response['compile_error'] = $details['compile_error'];
         }
+        if (is_null($contest) || $contest->partial_score || $run->score == 1) {
+            $problemsetProblem = \OmegaUp\DAO\ProblemsetProblems::getByPK(
+                $submission->problemset_id,
+                $submission->problem_id
+            );
+            if (!is_null($problemsetProblem)) {
+                $details['contest_score'] = $details['score'] * $problemsetProblem->points;
+            }
+        } else {
+            $details['contest_score'] = 0;
+            $details['score'] = 0;
+        }
```

Comment:
esto no se puede únicamente con el `else` dejando `contest_score` sin tocar? o sea

```suggestion
        if (!is_null($contest) && !$contest->partial_score && $run->score < 1) {
            $details['contest_score'] = 0;
            $details['score'] = 0;
        }
```

Reply:
Parece que no se está calculando correctamente el `contest_score` en uno de los casos y por ende, se obtiene el `score`  sin tomar en cuenta los puntos que se asignan al problema dentro de un `problemset`. Así que fallan un par de pruebas:
```
1) ContestUpdateTest::testCreateContestWhenPartialScoreIsUpdated with data set #2 (false, 100, 0, 5)
Failed asserting that 0.05 matches expected 5.0.

/opt/omegaup/frontend/tests/controllers/ContestUpdateTest.php:1207
/opt/omegaup/frontend/tests/controllers/ContestUpdateTest.php:1137

2) ContestUpdateTest::testCreateContestWhenPartialScoreIsUpdated with data set #3 (true, 100, 5, 0)
Failed asserting that 0.05 matches expected 5.0.
```

 y la única forma que se me ocurre de solucionarlo era agregando esa condición.

Reply:
ah ya, hay que ver cómo solucionarlo bien entonces. Eso se está estableciendo en https://github.com/omegaup/omegaup/blob/afd72326e6cd76b2aa92ae1ec5ebf85b8a3e315a/frontend/tests/Utils.php#L91

que debería ser `$run->contest_score` (y arriba debería ser `$run->score` por consistencia).

Reply:
Ah, con razón. No había encontrado donde arreglarlo y se me ocurría parcharlo desde ahí. Ya con ese cambio pasa las pruebas. Gracias.